litefs/getting-started-docker.html.md.erb

```diff
@@ -0,0 +1,139 @@
+---
+title: Getting Started with LiteFS in Docker
+layout: docs
+sitemap: false
+nav: litefs
+toc: true
+---
+
+## Overview
+
+This guide will walk you through the steps of getting a LiteFS cluster up and
+running using Docker. You can also use this guide as a reference for running
+LiteFS on a Linux server with minimal changes.
+
+For a full, working example of a LiteFS application, with a docker-compose setup
+that you can run locally, please see the [litefs-example][] repository.
+
+[litefs-example]: https://github.com/superfly/litefs-example
+
+## Creating a LiteFS Cloud Cluster (optional)
+
+<%= partial "/docs/reference/litefs/create_litefs_cloud_cluster" %>
+
+
+## Adding LiteFS to your Dockerfile
+
+<%= partial "/docs/reference/litefs/adding_litefs_to_your_dockerfile" %>
+
+## Configuring LiteFS
+
+Most configuration options for LiteFS are set via a YAML configuration file
+called `litefs.yml`. This file is typically placed in `/etc/litefs.yml` but
+you can change the path by using the `-config` flag.
+
+You can take a look at complete examples of what your `litefs.yml` file should
+look like for [the primary node][LiteFS primary yml] and [the replica nodes][LiteFS replica yml].
+
+[LiteFS primary yml]: https://github.com/superfly/litefs-example/blob/main/docker-config/etc/litefs.primary.yml
+[LiteFS replica yml]: https://github.com/superfly/litefs-example/blob/main/docker-config/etc/litefs.replica.yml
```

Comment:
You can (it works [here](https://github.com/superfly/docs/pull/836/files#diff-45ab410aceea9303b8f17bbe60750cc6aebb43ead2b6300c3e2ad0d86abea979R37) for example). I don't know what's going on here, but it may be related to the weirdness below? I will look into it in any case. 🙂